crates/matrix-sdk-ui/Cargo.toml

```diff
@@ -11,6 +11,8 @@ e2e-encryption = ["matrix-sdk/e2e-encryption"]
 native-tls = ["matrix-sdk/native-tls"]
 rustls-tls = ["matrix-sdk/rustls-tls"]
 
+experimental-uniffi = ["dep:uniffi"]
```

Comment:
I would be happy to assume that everything inside `matrix-sdk-ui` is experimental as we talked previously, and consider it's fine to get `uniffi` always present. Thoughts?

Reply:
Well, it has to be optional, otherwise non-FFI users of `matrix-sdk-ui` compile uniffi for no reason. I don't care that much whether it's called `experimental-`, but I think we'll have a `experimental-uniffi` feature in `matrix-sdk` sooner or later so maybe it makes sense for consistency?